ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+
+### Group Variables
+Group variables are defined in YAML files inside the group_vars folder:
+
+![Figure: Ansible Group Variables](../_media/getting-started/Group-vars-folder-structure.png)
+
+Please note that the example above is taken from the ansible-avd-cloudvision-demo repository. 
+
+Each file in the `group_vars` folder controls the variables for one of the groups defined in the `inventory.yml` file,
+so settings for the whole network can be specified in `DC1.yml`, and later overridden in `DC1_SPINES.yml` for just the hosts inside that group.
+
+A subset of DC1.yml is shown below:
+
+```
+# local users
+local_users:
+  admin:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+  cvpadmin:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+  ansible:
+    privilege: 15
+    role: network-admin
+    sha512_password: "<password-hash>"
+
+# Cloud Vision server information
+cvp_instance_ip: <CVP IP address>
+cvp_ingestauth_key: '<key>'
+
+# OOB Management network default gateway.
+mgmt_gateway: <IP of default gateway>
+
+# dns servers.
+name_servers:
+  - <IP of name server #1>
+  - <IP of name server #2>
+
+# NTP Servers IP or DNS name, first NTP server will be prefered, and sourced from Managment VRF
+ntp_servers:
+  - 0.<local country>.pool.ntp.org
+  - 1.<local country>.pool.ntp.org
+```
+
+Below is a subset of the DC1_FABRIC file:
+
+```
+# Leaf switch groups
+# A maximum of two nodes can form a leaf group
+# When two nodes are in a leaf group this will automatically form mlag pair
+
+l3leaf:
+  defaults:
+    # virtual router mac for VNIs assigned to Leaf switches
+    # format: xx:xx:xx:xx:xx:xx
+    virtual_router_mac_address: 00:1c:73:00:dc:01
+    platform: vEOS-LAB
+    bgp_as: 65100
+    spines: [DC1-SPINE1, DC1-SPINE2]
+    uplink_to_spine_interfaces: [Ethernet1, Ethernet2]
+    mlag_interfaces: [Ethernet3, Ethernet4]
+    spanning_tree_priority: 4096
+    spanning_tree_mode: mstp
+  node_groups:
+    DC1_LEAF1:
+      bgp_as: 65101
+      nodes:
+        DC1-LEAF1A:
+          id: 1
+          mgmt_ip: 10.255.0.13/24
+          spine_interfaces: [Ethernet1, Ethernet1]
+        DC1-LEAF1B:
+          id: 2
+          mgmt_ip: 10.255.0.14/24
+          spine_interfaces: [Ethernet2, Ethernet2]
+    DC1_LEAF2:
+      bgp_as: 65102
+      nodes:
+        DC1-LEAF2A:
+          id: 3
+          mgmt_ip: 10.255.0.15/24
+          spine_interfaces: [Ethernet3, Ethernet3]
+        DC1-LEAF2B:
+          id: 4
+          mgmt_ip: 10.255.0.16/24
+          spine_interfaces: [Ethernet4, Ethernet4]
+```
+
+The above defines a number of defaults that applies to all children of DC1_FABRIC along with more specific configuration
+for the DC1_LEAF1 group and even more specific configuration for the DC1-LEAF1A and DC1-LEAF1B switches.
+
+Group variables can be overridden by specifying host variables for specific devices.
```

Comment:
Please rephrase to remove redundancy. Suggestion: GV can be overridden by setting host variables for specific devices.